lexibank_ids.py

```diff
@@ -107,20 +114,54 @@ def cmd_makecldf(self, args):
         ids_lgs = [lg for lg in self.ids_raw_read("lang") if lg.status == "0"]
         ids_lgs_ids = [lg.lg_id for lg in ids_lgs]
 
+        edited_data = {}
```

Comment:
I think this block (and the subsequent blocks dealing with language/personnel names/IDs) could benefit from some comments that spell out the assumptions they make about the structure of the original XLSX files.